app/views/shipit/commits/_commit.html.erb

```diff
@@ -17,6 +17,15 @@
       <p class="commit-meta">
         <%= timeago_tag(commit.committed_at, force: true) %>
       </p>
+      <p class="commit-meta">
+        <% if commit.rollbackable%>
+          <span class='rollbackable-yes'> Safe to rollback </span>
+        <% elsif commit.rollbackable == false %>
+          <span class='rollbackable-no'> Unsafe to rollback </span>
+        <% else %>
+          <!-- Do not display anything if rollbackable is nil -->
```

Comment:
Tri-state boolean tend to be a smell. Why would a commit be `rollbackable=nil` ? Can't we default to either true or false ?

Reply:
Although we can treat `nil` functionally the same as the `true` case, I don't want to surface it to the ui if it's not explicitly set. i.e. if a merge is done without going through the merge queue flow, we won't have this value, or for apps that don't have the merge queue enabled. I could maybe convert this to `unsafe_to_rollback` so that at least the `false` and `nil` cases can be used interchangeably for most of the logic?